Gemfile-rails-dependencies

```diff
@@ -13,37 +13,22 @@ when /master/
   gem 'i18n', :git => 'https://github.com/svenfuchs/i18n.git', :branch => 'master'
   gem 'sprockets', :git => 'https://github.com/rails/sprockets.git', :branch => 'master'
   gem 'sprockets-rails', :git => 'https://github.com/rails/sprockets-rails.git', :branch => 'master'
-  if RUBY_VERSION  >= "2.2"
-    gem 'puma', :git => 'https://github.com/puma/puma', :branch => 'master'
-  end
+  gem 'puma', :git => 'https://github.com/puma/puma', :branch => 'master'
 when /stable$/
   gem_list = %w[rails railties actionmailer actionpack activerecord activesupport]
-  gem_list << 'activejob'  if version > '4-1-stable'
-  gem_list << 'actionview' if version > '4-0-stable'
-  if RUBY_VERSION  >= "2.2"
-    gem_list << 'puma' if version > '5-0-stable'
-  end
+  gem_list << 'activejob'  if version >= '4-2-stable'
+  gem_list << 'actionview' if version >= '4-2-stable'
```

Comment:
No need for the `if'`s

Reply:
As the minimum version is 4.2, these don't need an `if` statement anymore right?